pkg/webhook/pod/tikv_creater.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pod
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/pdapi"
+	"github.com/pingcap/tidb-operator/pkg/webhook/util"
+	admission "k8s.io/api/admission/v1"
+	core "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+const (
+	tikvNotBootstrapped = "\"TiKV cluster not bootstrapped, please start TiKV first\"\n"
```

Comment:
Use backquote to avoid escape the quote.

Reply:
fix